src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
 }
 
 @main(s"Opportunity ${opportunity.id.id}") {
-    <div class="grid-row">
-        <a href="@controllers.routes.OpportunityController.showOpportunities()">&lt; Return to opportunity list</a>
-        <p></p>
-        <h1>@opportunity.title</h1>
+<div class="grid-row">
 
+    <div class="column-full">
+        <p>
+            <a href="@controllers.routes.OpportunityController.showOpportunities()">&lt; Return to opportunity list</a>
+        </p>
+    </div>
+
+    <div class="column-two-thirds">
         <header class="page-header">
-            <table>
-                <tbody>
-                @titleCols.map { case (col1, col2) =>
-                <tr>
-                    <td>@Html(sectionLink(col1._1, col1._2))</td>
-                    <td>@col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
-                </td>
-                </tr>
-                }
-                </tbody>
-            </table>
+            <h1 class="heading-xlarge">@opportunity.title</h1>
         </header>
 
+        <aside>
+            <nav role="navigation" class="contain-floats page-navigation">
+                <ol>
+                    @titleCols.map { case (col1, col2) =>
+                        <li>
+                            @Html(sectionLink(col1._1, col1._2))
+                        </li>
+                    }
+                </ol>
+                <ol>
+                    @titleCols.map { case (col1, col2) =>
+                        <li>
+                            @col2.map { t => @Html(sectionLink(t._1, t._2)) }.getOrElse("")
+                        </li>
+                    }
+                </ol>
+            </nav>
+        </aside>
+
+
         <div>
-            <h2>Apply for funding</h2>
-            <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an application</a>
-            <p></p>
-            <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on seminars</a>
-            <p></p>
+            <h2 class="heading-large">Apply for funding</h2>
```

Comment:
I'd say this section is better off in the sidebar (it's not in the design so I'm free to have an opinion ;) )